test/module.test.js

```diff
@@ -613,6 +613,27 @@ describe('Test sidekick module', () => {
     assert.ok(notification.className.includes('modal-share-success'), 'Did not copy sharing URL to clipboard');
   }).timeout(IT_DEFAULT_TIMEOUT);
 
+  it('Displays page modified info on info button click', async () => {
+    const { checkPageResult, plugins } = await new SidekickTest({
+      page,
+      loadModule: true,
+      post: (p) => p.evaluate(() => window.hlx.sidekick
+        .shadowRoot
+        .querySelector('.hlx-sk .feature-container .info.plugin > button')
+        .click()),
```

Comment:
You can specify the plugin to be clicked like this:
```suggestion
      plugin: 'info',
```

Note: due to the asynchronous status fetching, you may need to add a `pluginSleep` value.